From 483273aebf9fdae0b12a7615312956cd016a273a Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 5 Jan 2024 15:46:06 +0100
Subject: [PATCH 0371/1695] dts: stm32: fix TAMP node on stm32mp131.dtsi

To handle subnodes, TAMP needs #address-cells and #size-cells.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9f783f423bb61c21da86de9831c42bd31d542eaa dts: stm32: fix TAMP node on stm32mp131.dtsi]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I5e9e61aeca17edeb5f6426e44d1d975b2c35304f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349581
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp131.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp131.dtsi b/core/arch/arm/dts/stm32mp131.dtsi
index b7347f41b..2b6bcbdc6 100644
--- a/core/arch/arm/dts/stm32mp131.dtsi
+++ b/core/arch/arm/dts/stm32mp131.dtsi
@@ -329,6 +329,8 @@
 		tamp: tamp@5c00a000 {
 			compatible = "st,stm32mp13-tamp";
 			reg = <0x5c00a000 0x400>;
+			#address-cells = <1>;
+			#size-cells = <1>;
 			interrupts = <GIC_SPI 128 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc RTCAPB>;
 		};
-- 
2.47.2

